oat-sa/tao-core-ui-fe
# Feature/CON-185/Support pages in resource manager

Related to: https://oat-sa.atlassian.net/browse/CON-185
https://github.com/oat-sa/tao-core/pull/2747
 oat-sa/extension-tao-item#445
 oat-sa/extension-tao-mediamanager#245

Added pagination in resource manager.

How to test:
- go to Item Authoring
- add Image to item
- check resource manager
